save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestSuitesSourceService.kt

```diff
@@ -89,7 +91,7 @@ class TestSuitesSourceService(
     @Transactional
     fun update(entity: TestSuitesSource): Boolean {
         requireNotNull(entity.id) {
-            "Cannot update entity as it is not saved yet: $this"
+            "Cannot update entity as it is not saved yet: $entity"
```

Comment:
Entity classes don't have an overridden `toString`, so we should probably just add necessary fields manually

Reply:
changed to `name` and `organizationName`